devel/patch/patches/patch-tests_Makefile.am

```diff
@@ -0,0 +1,14 @@
+$NetBSD$
```

Comment:
Will this actually have any effect? It does not look as if you re-run `automake`. Maybe this change needs to be in `Makefile.in`?

Reply:
Doh, of course, sorry for the confusion. Fixed